controllers/game_controller.rb

```diff
@@ -0,0 +1,62 @@
+class GameController < BaseController
+  def get
+    render
+  end
+
+  def post
+    proceed_user_params
+    return proceed_finished_game if game.win? || game.lose?
+
+    render
+  end
+
+  private
+
+  def proceed_user_params
+    if @request.params.key?('hint')
```

Comment:
string here should be in constant or in I18n

Reply:
this string for application use, not user-read. Probably other strings should be localized, but not this

Reply:
This kind of magic string. If you're not going to translate then you can use constant to save it